### PR TITLE
React-Calculator milestone-2 

### PR DESCRIPTION
**Pull request by Wusinho**
This project is an App Calculator using React.
## Built with

- HTML
- JavaScript
- Webpack
- Package.json
- React

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ButtonPanel from './ButtonPanel';
+import '../assets/styles/App.scss';
+
+const App = () => (
+  <div>
+    <ButtonPanel />
```
Comment (afanimartin) [inline src/components/App.js:10]: - You need to also render the `Display` component
- Use `React.Fragment` shorthand syntax: `<>...</>`

File: src/components/Display.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Rectangle = (props) => (<div className="rectangle">{props.value || '0'}</div>);
```
Comment (afanimartin) [inline src/components/Display.jsx]: - Why is the component named `Rectangle`? It's not a clear name. Change it to `Display`

File: src/components/Square.jsx [hunk 3]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/button-has-type */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Square = (props) => (<button className="square" onClick={props.onClick}>{props.value}</button>);
```
Comment (afanimartin) [inline src/components/Square.jsx]: - This name doesn't make sense. It should be `Button`. The component is a button. Make it clear.

File: src/components/Square.jsx [hunk 4]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable react/destructuring-assignment */
```
Comment (afanimartin) [inline src/components/Square.jsx]: - Do not disable the linter rules. Learn to fix those errors instead. In this case, the errors can actually be fixed.

File: README.md [hunk 5]
```diff
@@ -2,9 +2,9 @@
 
 ## About
 
-This project is an App Calculator using React
+This project is an App Calculator using React.
 
-<!-- ![screenshot](./src/assets/Todo-list.png) -->
+![screenshot](./src/assets/static/Screenshot.png)
```
Comment (afanimartin) [inline README.md:7]: - Update the `To get a local copy of this repo` instruction. Step 1 has this instruction: `$ cd "here comes the path of your selected directory (double quotes must not be here)"`. This instruction is not clear and doesn't look useful to a user who wants to checkout the app locally. Always be very explicit in your instructions.

**Review by afanimartin — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Nice work in this milestone.
### Highlights 👍🏿 👍🏿 👍🏿 
- Correct implementation of `ButtonPanel` component

### Change(s) 🔕 🔕 🔕 
- Update readme file. Check comments
- Check comments for more suggestions


### Martin Afani
#### Slack: whiz25

**Review by taiwo2 — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED ♻️ 
@Wusinho well done for the task, Kindly implement the changes suggested by the previous Code Reviewer:
### Highlights
- [ ] Use React.Fragment shorthand syntax: <>...</> 👍🏻 
#### To Improve
### `App` component
- [ ] it renders the `Display` and `ButtonPanel` components  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/appFunctionalComponent.png))
###  `button` component
- [ ] Do not disable the linter rules. Learn to fix those errors instead. In this case, the errors can actually be fixed in your `button.jsx`
- [ ]   It should be Button. The component is a button. Make it clear.
- [ ]  it accepts `a button name` as an incoming prop ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/button/acceptProp.png))

### Once specific changes make kindly request for another review

**Review by NtwaliHeritier — APPROVED** [review]
### **STATUS: APPROVED👍️👍️👍️👍️**

Hello @Wusinho 
Great work making the requested changes👏️👏️👏️

You can merge👍️👍️

Happy coding💻️💻️

**Pull request merged**